root/frontend/src/components/Weather/Weather.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import useStyles from './Weather.style';
+
+function Weather() {
+  const classes = useStyles();
+  const fetchWeatherData = async (): Promise<any> => {
+    const data = await axios('http://localhost:5005/weather/getWeatherInfo');
+    return data.data;
+  };
+
+  type WeatherDataObj = {
+    current: any;
+    location: any;
+  };
+
+  const [weatherData, setWeatherData] = useState<WeatherDataObj>();
+
+  useEffect(() => {
+    const getWeather = async () => {
+      const fetchedWeatherData = await fetchWeatherData();
```

Comment:
I think we should probably add some more linting plugins to catch for `useEffect()` not having all dependencies listed (in this case `fetchWeatherData()`). See here for more info:
* https://reactjs.org/docs/hooks-rules.html
* https://www.npmjs.com/package/eslint-plugin-react-hooks

Let me know what you think if we should or shouldn't have this plugin.